docker/hub/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd $(git rev-parse --show-toplevel)
```

Comment:
Magic and voodoo :( I'd prefer if we can avoid needing to know the working directory at all, but I guess we need to to specify the dockerfiles later on or something...

Reply:
Yeah, this is one of those annoying "`docker build` makes the build context out of the current working directory" things. :-(